src/test/java/net/openhft/chronicle/threads/MediumEventLoopTest.java

```diff
@@ -70,6 +70,20 @@ public boolean action() {
         }
     }
 
+    @Test
+    void concurrentStartStopDoesNoThrowError() throws ExecutionException, InterruptedException {
+        ExecutorService es = Executors.newCachedThreadPool();
+        for (int i = 0; i < 100; i++) {
+            try (final MediumEventLoop mediumEventLoop = new MediumEventLoop(null, "test", Pauser.balanced(), false, "any")) {
+                final Future<?> starter = es.submit(mediumEventLoop::start);
+                final Future<?> stopper = es.submit(mediumEventLoop::stop);
+                starter.get();
+                stopper.get();
+            }
+        }
+        ExecutorServiceUtil.shutdownAndWaitForTermination(es);
+    }
```

Comment:
If you get rid of the `startStopMutex` this reliably reproduces the `RejectedExecutionException`